configs/sst_security_crypto-openssl-c9s.yaml

```
@@ -0,0 +1,15 @@
+---
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: openssl
+  description: >-
+    General purpose cryptographic library and toolkit for TLS and SSL protocols.
+  maintainer: sst_security_crypto
+
+  packages:
+    - openssl
+    - compat-openssl11
+
+labels:
+  - c9s
```

Indenting needs to be fixed:
```suggestion
  labels:
    - c9s
```

---

Yeah, I'm looking at that, and looking at a PR for content-resolver to improve the error message.